scripts/pre_commit_linter.py

```diff
@@ -177,6 +177,9 @@
         'regexp': re.compile(r'\b(browser.sleep)\('),
         'message': 'In tests, please do not use browser.sleep().',
         'excluded_files': (
+            # TODO(#7622): Remove the file from the excluded list.
```

Comment:
Could you please add, to this TODO: "Remove the TODO in core/tests/protractor_desktop/embedding.js pointing to the same issue"?

Reply:
done!